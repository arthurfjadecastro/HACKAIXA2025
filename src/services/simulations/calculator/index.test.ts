import * as calculatorIndex from './index';

describe('Simulations Calculator Index', () => {
  it('should export calculator modules', () => {
    expect(calculatorIndex).toBeDefined();
    expect(typeof calculatorIndex).toBe('object');
  });

  it('should re-export calculator functions without errors', () => {
    // Verifica se mÃ³dulos foram exportados sem erros
    expect(() => {
      const exported = { ...calculatorIndex };
      expect(exported).toBeDefined();
    }).not.toThrow();
  });

  it('should match snapshot', () => {
    expect(Object.keys(calculatorIndex).sort()).toMatchSnapshot();
  });
});
