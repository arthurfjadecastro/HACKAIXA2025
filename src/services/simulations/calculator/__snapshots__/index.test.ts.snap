// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simulations Calculator Index should match snapshot 1`] = `
[
  "calculateLoanSchedule",
  "convertMonthlyToYearlyRate",
  "convertYearlyToMonthlyRate",
]
`;
